Source/Renderer/UniformState.js

```diff
@@ -203,6 +205,19 @@ define([
 
         /**
          * @memberof UniformState.prototype
+         * @type {Texture}
+         */
+        globeDepthTexture : {
```

Comment:
Just replace this with a regular property.  It will be slightly faster and more concise.